persim/visuals.py

```diff
@@ -17,7 +17,10 @@ def plot_diagrams(
     lifetime=False,
     legend=True,
     show=False,
-    ax=None
+    ax=None,
+    save=False,
+    figures_directory=None,
+    filename=None
```

Comment:
Do you need both `save` and `filename`?  How about if `filename is not None`, then save to the `filename`